libs/server/controllers/user/src/lib/user.service.ts

```diff
@@ -235,7 +235,7 @@ export class UserService {
         id,
       },
       data: {
-        profilePicture: `${file.filename}`,
+        profilePicture: `${MulterService.destinations.profilePictures}/${file.filename}`,
```

Comment:
gibt MulterService.destinations.profilePicture da ganze path zrück oder nur /profilepicture oder so ?

Reply:
beispiel für an entry in da datenbank: `"uploads/profile-pictures/file.jpg"`